<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Research</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>






<link rel="stylesheet" href="assets/styles.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Claire Winfrey</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="Research.html">Research</a>
</li>
<li>
  <a href="Publications.html">Publications</a>
</li>
<li>
  <a href="Outreach.html">Outreach</a>
</li>
<li>
  <a href="cv.html">CV</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Research</h1>

</div>


<style>
  /* this just styles the big page title */
  #header h1.title {
    text-align: center;
    font-size: 2.5rem !important;
    margin-top: 0.5rem;
  }
</style>
<div id="dissertation-research" class="section level2">
<h2>Dissertation Research</h2>
<style>
.centered-img {
  display: block;
  margin: 1rem auto;   /* centers horizontally */
  max-width: 1000px;   /* max width */
  width: 100%;         /* scales down on small screens */
  height: auto;
}
</style>
<figure class="centered-figure">
<img src="assets/img/TyvekTruckSRS_2.jpg" alt="Claire and a truck" class="centered-img">
<figcaption style="font-weight:600; font-size:1.15rem; line-height:1.25;">
Most of my dissertation field research took place in the Corridor
Project experimental landscapes at the Savannah River Site, SC, and Iâ€™m
grateful to Team Corridor and the U.S. Forest Service for making it
happen! Photo: Andrea Torres
</figcaption>
</figure>
<div
id="habitat-specialization-and-edge-effects-of-soil-microbial-communities-in-a-fragmented-landscape-winfrey-resasco-fierer-2025"
class="section level3">
<h3>Habitat specialization and edge effects of soil microbial
communities in a fragmented landscape <a
href="https://esajournals.onlinelibrary.wiley.com/doi/full/10.1002/ecy.70072">(Winfrey,
Resasco &amp; Fierer, 2025)</a></h3>
<figure class="figure-left">
<img src="assets/img/EU8patchmatrix.jpg" alt="picture of fieldwork site">
<figcaption style="font-weight:600; font-size:1.15rem; line-height:1.25;">
Open patch with forested matrix in background (Savannah River Site,
South Carolina)
</figcaption>
</figure>
<p>While edge effects caused by habitat fragmentation have been well
documented in plants and animals, whether or not soil bacteria, archaea,
and fungi exhibit edge effects is an open question. Given the central
role of microorganisms as drivers of soil ecosystem services, examining
the responses of soil microorganisms to edge effects is important
initial step for understanding how soil functioning may be impacted by
habitat fragmentation.</p>
<p>Motivated by this knowledge gap, we used 16S and ITS rRNA gene
sequencing to survey the spatial distributions of soil bacteria,
archaea, and fungi communities in replicate ~1 hectare open patches
surrounded by forested plantation matrix in South Carolina, USA. We
discovered that the greater the vegetation contrast between the patch
and the matrix, the more pronounced was the microbial turnover across
the edge and more distinct were patch and matrix communities. These
observed edge effects were driven mostly by taxa that specialized in the
open patch or the forested matrix; there were very few taxa that
preferred edge habitat. We next explored if phylogeny or life history
traits might distinguish taxa that responded to the edge from generalist
taxa. We did not find support for our hypotheses that generalist taxa
would be more likely to sporulate or have larger genomes (associated
with greater metabolic versatility). However, we did find that
Chloroflexi and arbuscular mycorrhizal fungi strongly preferred the open
patch over matrix or edge habitat, suggesting that they may be
particularly vulnerable to fragment edges.</p>
</div>
<div
id="the-sources-and-traits-of-bacteria-and-fungi-found-in-the-near-surface-atmosphere.-paper-to-be-submitted-soon"
class="section level3">
<h3>The sources and traits of bacteria and fungi found in the
near-surface atmosphere. (Paper to be submitted soon!)</h3>
<figure class="figure-left">
<img src="assets/img/andreaLookingInWonderUPAS.jpg" alt="picture of Andrea Torres">
<figcaption style="font-weight:600; font-size:1.15rem; line-height:1.25;">
REU student Andrea Torres and bioaerosol sampler above! (Savannah River
Site, South Carolina)
</figcaption>
</figure>
<p>High UV, rapidly fluctuating temperatures, and few nutrients make the
atmosphere a harsh place for microorganisms. Yet, diverse fungi,
bacteria, archaea, and protists are found in the near-surface atmosphere
(i.e., the â€˜aeromicrobiomeâ€™), with transport through the atmosphere long
hypothesized to explain the continental or even global ranges of
microbes. This project asks three unresolved, key questions in
aeromicrobiology, working at habitat boundaries (where savanna-like
patches and a forest matrix meet) and a spatial scale of ~100 meters to
investigate the influence of local plants and soils on the
aeromicrobiome. A better understanding of the aeromicrobiome at
ecosystem edges may lead to improved predictions for microbial dispersal
and spillover in the context of widespread and increasing habitat
fragmentation worldwide.</p>
<p>Questions I addressed in this project: (1) Is the plant community
type important for driving differences in the aeromicrobiome at small
scales? (2) Which, if any, microbes are shared between the air and local
plants and soils? (3) Are there microbial traits that distinguish
aerosolized microbes from those in the soil or plant source
environments?</p>
</div>
<div
id="the-diversity-and-distribution-of-soil-bacteria-capable-of-forming-spore-like-structures.-in-preparation"
class="section level3">
<h3>The diversity and distribution of soil bacteria capable of forming
spore-like structures. (in preparation)</h3>
<p>Environmental characteristics that shape bacterial communities, such
as pH, moisture, and the availability of carbon and nutrients, are
highly patchy both temporally and spatially in soil. How do soil
bacteria survive harsh conditions where they live or survive the journey
to patches of soil where conditions are better? Ample research has shown
that some bacterial taxa armor themselves in tough spore-like structures
in response to conditions unfavorable for growth. However, such a
strategy should be adaptive for more than just these few phyla that we
know independently evolved and form different types of spores.</p>
<p>In this project, I address three questions: (1) which soil bacteria
form spore-like structures? (2) are these identified spore-forming
bacteria more ubiquitous than non-spore-forming bacteria in soil? (3)
How do spore-forming bacteria vary across ecosystems and environmental
gradients? And (4) are spore-forming soil bacteria more readily
dispersed in the near-surface atmosphere? To address these questions, I
combine a culture-independent enrichment approach and marker-gene
sequencing to classify putative spore-formers and non-spore formers
directly in their soil environment.</p>
</div>
</div>
<div id="other-research" class="section level2">
<h2>Other Research</h2>
<div class="img-row">
<p><img src="assets/img/Ecivile.png" alt="Enallagma civile marked in the field">
<img src="images/pic2.png" alt="pic2">
<img src="images/pic3.png" alt="pic3"></p>
</div>
<style>
.img-row {
  display: flex;
  flex-wrap: wrap;      /* allow wrapping on narrow screens */
  gap: 10px;            /* space between images */
  justify-content: center;
}

.img-row img {
  flex: 1 1 200px;      /* grow/shrink with min width ~200px */
  max-width: 100%;      /* donâ€™t overflow container */
  height: auto;         /* keep aspect ratio */
  border-radius: 6px;   /* optional: rounded corners */
}
</style>
<div
id="do-biogeographical-rules-apply-to-microorganisms-dickey-et-al.-2021"
class="section level3">
<h3>Do biogeographical â€˜rulesâ€™ apply to microorganisms? <a
href="https://www.frontiersin.org/journals/ecology-and-evolution/articles/10.3389/fevo.2021.633155/full">(Dickey
et al., 2021)</a></h3>
<p>Macroecological â€˜rulesâ€™ are a series of patterns that essentially
summarize our current understanding of biogeography. Based on centuries
of observations of animals and plants, these â€˜rulesâ€™ describe the
relationships between distribution on the Earthâ€™s surface (e.g.,
latitude, area, temperature) to organismal traits or community assembly
and biodiversity. However, our knowledge of if these rules apply to the
majority of species on Earthâ€“the largely unseen bacteria, archaea, and
fungiâ€“is only now starting to become clear. In this paper, we perform a
literature meta-analysis to assess how well microorganisms conform to
biogeographical patterns, considering aquatic and terrestrial systems as
well as host-assoicated and free-living microbes. Along with Anchal
Padukone, I wrote the Island Biogeography section!</p>
</div>
<div
id="do-male-damselflies-males-use-senses-other-than-visual-to-detect-their-mates-winfrey-fincke-2017"
class="section level3">
<h3>Do male damselflies males use senses other than visual to detect
their mates? <a
href="https://www.tandfonline.com/doi/abs/10.1080/13887890.2017.1297259">(Winfrey
&amp; Fincke, 2017)</a></h3>
<p>Because damselflies are speciose, have radiated recently, and have
complex mating behaviors, damselflies are model organisms for studies of
sexual selection and sexual conflict. Most of this work to date operates
under the assumption that males use only visual cues to detect and
identify mates. Our research was the first to test this assumption in
the field, finding that males do not orient to females without visual
cues.</p>
</div>
<div
id="how-prevalent-is-batrachochytrium-dendrobatidis-bd-among-amphibians-in-southern-oklahoma-marhanka-et-al.-2017"
class="section level3">
<h3>How prevalent is Batrachochytrium dendrobatidis (Bd) among
amphibians in Southern Oklahoma? <a
href="https://cameronsiler.com/detection-of-high-prevalence-of-batrachochytrium-dendrobatidis-in-amphibians-from-southern-oklahoma/">(Marhanka
et al.Â 2017)</a></h3>
<p>Batrachochytrium dendrobatidis (Bd, often called chytrid for short)
has caused massive declines in amphibian populations worldwide. Research
supports that the prevalence and lethality of Bd worldwide is on the
rise, due to human-mediated spread of the fungus and warmer global
temperatures, respectively. However, despite the threat of this fungus
and Oklahomaâ€™s high amphibian biodiversity, only 4 isolated sites had
been sampled for Bd in Oklahoma prior to this work. For this project, we
sampled 14 sites across Southern Oklahoma for frogs and salamanders. All
amphibian families, all sampling locations, and 15/18 species were Bd +,
although locations and species differed in their pathogen loads.</p>
</div>
<div
id="describing-a-new-species-of-philippine-slender-skink-brachymeles-bonitae-complex-davis-et-al.-2016"
class="section level3">
<h3>Describing a new species of Philippine Slender Skink (Brachymeles
bonitae complex) <a
href="https://cameronsiler.com/additions-to-philippine-slender-skinks-of-the-brachymeles-bonitae-complex-iii/">(Davis
et al.Â 2016)</a></h3>
<p>Philippine skinks of the genus Brachymeles are a fascinating group
with varying degrees of limb and digit loss that are thought to help
them adapt to specific challenges in their environment. Work lead by
Dr.Â Cameron Siler has discovered many cryptic species within
Brachymeles, with additional evidence that more cryptic species are
present within the B. bonitae complex. This project reports formal
description of a new species of fossorial skink within the Brachymeles
bonitae complex found on Tablas Island in the Philippines, based on
collections made over 50 years ago. This work is near and dear to my
heart, as it was the first scientific paper I contributed to. I
appreciate Dr.Â Siler giving me and the other undergraduates on this
project an opportunity to publish!</p>
</div>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
